src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -36,7 +36,6 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
     waitAndCloseProject()
   }
 
-  @Ignore // FIXME (#830): ui test failing - socket timeout
```

Comment:
Wow actually... so your changes from this PR are enough to fix all the flakiness? 😯

Reply:
I.e. just selecting the right `IdeFrameImpl`